WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -179,6 +189,26 @@ public void onErrorResponse(VolleyError error) {
                 });
     }
 
+    private void fetchJetpackSettings() {
+        WordPress.getRestClientUtils().getJetpackSettings(
+                mSite.getSiteId(), new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        AppLog.d(AppLog.T.API, "Received response to Jetpack Settings REST request.");
```

Comment:
I don't think we do a very good job with this, but I think `AppLog.v` might be more appropriate here since it's probably too valuable during debug. I also find it interesting that you chose `AppLog.w` over `AppLog.e` while handling the error, I think it [has the lowest priority overall](https://developer.android.com/reference/android/util/Log.html).